tahowallet/taho.xyz
# updated site metadata and image on download page

Changes
* updated two images - metadata cover and `/download` hero
* updated metadata with correct twitter handle, meta description, meta title, and canonical URL

Test
* check https://taho.xyz/download/ page for image that changes wallet address to a .eth address vs .ens address
* try google searching `taho.xyz` or `taho wallet` to see if a description and updated meta image appear
